Documentation/user-manual.txt

```diff
@@ -229,12 +229,15 @@ of each dive. These panels are respectively marked [red]#A#, [red]#B# and
 in the figure below. Each of these tabs will now be explained for data entry.
 
 image::images/AddDive1_f22.jpg["FIGURE: Add dive",align="center"]
-When you edit a field in Notes or Equipment panels, _Subsurface_ enters *Editing Mode*,
+When you edit a field in the Notes or Equipment panels, _Subsurface_ enters *Editing Mode*,
```

Comment:
that's actually no longer true...
at this point we only have *Editing Mode* when you edit the Equipment panel.